local/query-genomic.sh

```diff
@@ -0,0 +1,3 @@
+curl -X POST "http://localhost:8080/sequence/genomic"  -H 'Content-Type: application/json' --no-buffer \
```

Comment:
Great that you have API tests.  this could use RESTAssured instead.  Also, i wonder if the test data should be in a dedicated `test/` directory or similar

Reply:
Agree with Steve that this should live in a "test" dir.  "local" can mean too many other things.